pytorch_lightning/plugins/precision/tpu_bfloat.py

```diff
@@ -26,6 +26,9 @@ class TPUHalfPrecisionPlugin(PrecisionPlugin):
 
     precision: int = 16
 
+    def __init__(self) -> None:
+        super().__init__()
+
```

Comment:
Why is this necessary?

Reply:
```
class PrecisionPlugin(Plugin):
    """ Plugin handling the precision-specific parts of the training.
    The static class attributes EPSILON and precision must be overwritten in child-classes and their
    default values reflect fp32 training.
    """
    EPSILON: float = 1e-6
    precision: Union[str, int] = 32

    def __init__(self) -> None:
        super().__init__()
        self.clip_grad_funcs = {
            GradClipAlgorithmType.VALUE: self.clip_grad_by_value,
            GradClipAlgorithmType.NORM: self.clip_grad_by_norm,
        }
```

`clip_gradient` functions use `self.clip_grad_funcs`. If we do not call `super().__init__()`, `self.clip_grad_funcs`is not initialized.

Reply:
But Python adds this empty `__init__` by default so it's redundant

Reply:
You're right. WIP

Reply:
Finished !